runtime/queries/just/injections.scm

```diff
@@ -0,0 +1,14 @@
+(comment) @comment
+
+(shebang_recipe 
+    (shebang 
+        interpreter:(TEXT) @language)
+    (shebang_body) @content
+) 
+
+(source_file 
+    (item (setting lang:(NAME) @language))
+    (item (recipe (body (recipe_body) @content)))
+) 
+
+; (interpolation (expression) @just)
```

Comment:
```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))

(shebang_recipe 
    (shebang 
        interpreter:(TEXT) @injection.language)
    (shebang_body) @injection.content
) 

(source_file 
    (item (setting lang:(NAME) @injection.language))
    (item (recipe (body (recipe_body) @injection.content)))
) 

; ((interpolation (expression) @injection.content)
;  (#set! injection.language "just"))
```

neovim has a custom way of doing injections, we use the same way as tree-sitter's highlight crate: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection